### PR TITLE
Update homebrew plugins for version 1.1

### PR DESCRIPTION
**Pull request by nak3**
As per title, this is a same patch with https://github.com/knative-sandbox/homebrew-kn-plugins/pull/45 but for v1.1.

### CODE REVIEW

File: quickstart@0.6.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require 'fileutils'
```
Comment (csantanapr) [inline quickstart@0.6.rb:1]: Why is this file name `@0.6.rb` instead of `@1.0.rb` ?

**Review by csantanapr — CHANGES_REQUESTED** [review]

File: quickstart@0.6.rb (hunk 1 above)
Comment (psschwei) [reply quickstart@0.6.rb:1]: Yes, this filename should be `quickstart@1.0.rb`

**Review by csantanapr — CHANGES_REQUESTED** [review]
fix kn quickstart file to 1.0

**Comment by csantanapr** [comment]
/hold
needs a fix to the file name for quickstart https://github.com/knative-sandbox/homebrew-kn-plugins/pull/55#discussion_r768910788

File: quickstart@0.6.rb (hunk 1 above)
Comment (nak3) [reply quickstart@0.6.rb]: Sure, updated.

**Comment by nak3** [comment]
Thank you @csantanapr @psschwei Updated.

**Pull request merged**